File: test/fixtures/r/get_with_env_var.r
library(httr)

headers = c(
  `Content-Type` = "application/json",
  Authorization = paste0("Bearer ", Sys.getenv("DO_API_TOKEN"))
)

params = list(
  type = "distribution"
)

res <- httr::GET(url = "http://localhost:28139/v2/images", httr::add_headers(.headers=headers), query = params)

File: test/fixtures/clojure/get_with_env_var.clj
(require '[clj-http.client :as client])

(client/get "http://localhost:28139/v2/images" {:query-params {:type "distribution"}
                                                :headers {"Content-Type" "application/json",
                                                          "Authorization" (str "Bearer " (str (System/getenv "DO_API_TOKEN")))}})
